packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -4266,6 +4266,7 @@ class InheritedElement extends ProxyElement {
   ///    created with [inheritFromWidgetOfExactType].
   ///  * [setDependencies], which sets dependencies value for a dependent
   ///    element.
+  ///  * [removeDependencies], which unsets the value for a dependent element.
   ///  * [notifyDependent], which can be overridden to use a dependent's
   ///    dependencies value to decide if the dependent needs to be rebuilt.
   ///  * [InheritedModel], which is an example of a class that uses this method
```

Comment:
these lists of "see also" links is getting out of hand. We don't need to mention every method in each one. We should clean it up so that only the most relevant ones are linked to each time.